### PR TITLE
Setting number_of_fragments to 0

### PR DESCRIPTION
**Pull request by muzaffaryousaf**

### CODE REVIEW

**Comment by muzaffaryousaf** [comment]
@symbolist Do we need to write unit tests for this ?

**Comment by symbolist** [comment]
@muzaffaryousaf Yes.

File: notesapi/v1/tests/test_views.py [hunk 1]
```diff
@@ -717,6 +717,57 @@ def test_search_highlight(self):
...
+            felis eu pede mollis pretium. Integer tincidunt. Cras dapibus. Vivamus elementum semper nisi. Aenean
+            vulputate eleifend tellus. Aenean leo ligula, porttitor eu, consequat vitae, eleifend ac, enim. Aliquam
+            lorema ante, dapibus in, viverra quis, feugiat a, tellus. Phasellus viverra nulla ut metus varius laoreet.
+            Quisque rutrum. Aenean imperdiet. Etiam ultricies nisi vel augue. Curabitur ullamcorper ultricies nisi.
+            Nam eget dui. Etiam rhoncus. Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libero,
+            sit amet adipiscing sem neque sed ipsum. Nam quam nunc, blandit vel, luctus pulvinar, hendrerit id, lorema.
+            Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis
+            ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh.
+            Donec sodales sagittis magna. Sed consequat, leo eget bibendum sodales,
+            augue velit cursus nunc, Lorem'''
```
Comment (symbolist) [inline notesapi/v1/tests/test_views.py]: This can be shortened to:
```
text = "Lorem " + "word " * 100 + "Lorem"
```

**Comment by muzaffaryousaf** [comment]
@symbolist Review ??

**Comment by symbolist** [comment]
:+1:

**Pull request merged**